src/request-sender.ts

```diff
@@ -1,5 +1,5 @@
 import { CookiesStatic } from 'js-cookie';
-import merge from 'lodash/merge';
+import merge from 'lodash-es/merge';
```

Comment:
Instead, I would rather us use the `lodash.merge` + types packages instead.

But on another note, is this really causing build performance issues?

Reply:
my understanding fro tree shaking was that using either the es version or the main version + the babel plugin were the best ways to go. since this package used only one function I went with lodash-es. happy to switch to lodash.merge

Reply:
Using the individual packages are essentially using their tree shaked version for the core library. In this case you are bringing in two lodash libraries (`lodash` and `lodash-es`) and letting the compiler "tree shake" which essentially `tsc` doesn't really do for external  dependencies. You'll need a bundler to actually tree shake external dependencies.

Reply:
right so our build is set up to tree shake, but I figured that there was no need for an entire lodash here which was the main reason I swapped it. When you use the entire lodash the best way to tree shake.

We use the plugins mentioned here https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark to help us out apart from the other things we do.

Reply:
Yeah, I'd rather us import what we only need (just the merge function) rather than an entire library thinking we might need it later. But thanks for providing this optimization for the library!